openff/interchange/smirnoff/_create.py

```diff
@@ -371,10 +371,20 @@ def _plugins(
 
         if len(handler_classes) == 1:
             handler_class = handler_classes[0]
-            collection = collection_class.create(
-                parameter_handler=force_field[handler_class._TAGNAME],
-                topology=topology,
-            )
+            try:
+                collection = collection_class.create(
+                    parameter_handler=force_field[handler_class._TAGNAME],
+                    topology=topology,
+                )
+            except TypeError:
+                collection = collection_class.create(
+                    parameter_handler=force_field[handler_class._TAGNAME],
+                    topology=topology,
+                    vdw_collection=interchange[
+                        "DoubleExponential"
+                    ],  # need to hook into .acts_as attribute
```

Comment:
This is going to be a pain, I'm afraid there will need to be logic along the lines of "if any collection is meant to act as a vdW collection, that takes precedence over a vanilla vdW collection"